rst2pdf/createpdf.py

```diff
@@ -1807,23 +1805,30 @@ def __init__(self):
             prefix = os.getcwd()
         if prefix not in sys.path:
             sys.path.insert(0, prefix)
+
         log.info('Importing extension module %s', repr(modname))
+
         firstname = path_given and modname or (modname + '_r2p')
         _names = [firstname, modname]
-        try:
-            for _name in _names:
-                try:
-                    module = import_module(_name, 'rst2pdf')
-                except ImportError:
-                    continue
-        except ImportError as e:
-            if str(e).split()[-1].replace("'", '') not in [firstname, modname]:
+        exc = None
```

Comment:
Can we call this variable `import_error` or similar?

Reply:
Sure